Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM ubuntu:18.04
+LABEL image=go-translate-server
+
```

Comment:
Use [multi-stage](https://docs.docker.com/develop/develop-images/multistage-build/) builds

Reply:
Thanks, now it's multi-stage: `builder` is on `golang:1.16`, `artifact` is on `ubuntu:20.04`.